api/config.go

```diff
@@ -24,37 +28,43 @@ var (
 
 // Config structure is a main config object
 type Config struct {
-	Version                 string
-	Revision                string
-	MesosIPDiscoveryCommand string
-	DCOSVersion             string
-	SystemdUnits            []string
+	Version                                      string `json:"-"`
```

Comment:
nit: prefer `omitempty` over `-` because we use `omitempty` in a lot of our other Mesosphere golang code.